### DISCUSSION TITLE
AWS support for multiple high throughput EFA interfaces

### DISCUSSION THREAD

**Discussion by flowinh2o** [skypilot-org/skypilot#1971] [Q&A]
Hello.  There are certain instance types that support multiple network cards such as the A100 p4d instance types that support up to 4 NICs.  Is it possible to setup a node with max support network cards and EFA enabled.  That way someone could take advantage of the 400Gb/s (4 x 100Gb) throughput between node for distributed GPU training?

**Comment by concretevitamin** [comment]
It should be possible. One starting point is to customize the `node_config` field in https://github.com/skypilot-org/skypilot/blob/master/sky/templates/aws-ray.yml.j2 to have something like: https://github.com/ray-project/ray/issues/33586. Note the `InterfaceType: efa` part.

We don't have expertise in this area and would love some community support here.
